### PR TITLE
Add support for MAIL_FROM property

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by nickanderson** [comment]
:/ I am not any kind of elisp expert. The change works for me, but I am not clear to me what needs to change to pass the test.

**Comment by nickanderson** [comment]
Ah, got it. 
fixed, squashed.

File: org-mime.el [hunk 1]
```diff
@@ -479,16 +480,20 @@ If SUBTREEP is t, curret org node is subtree."
     (message "Warning: org-element-map is not available. File keywords will not work.")
     '())))
 
-(defun org-mime-build-mail-other-headers (cc bcc)
-  "Build mail header from CC and BCC."
+(defun org-mime-build-mail-other-headers (cc bcc from)
+  "Build mail header from CC, BCC, and FROM."
```
Comment (redguardtoo) [inline org-mime.el]: You need better algorithm for this part. The original code might not be written by me.

**Comment by redguardtoo** [comment]
Here is my code to replace the body of `org-mime-build-mail-other-headers`,
```
(let* ((arr (list (cons "Cc" cc) (cons "Bcc" bcc)  (cons "From" from )))
       rlt)
  (dolist (e arr)
    (when (cdr e)
      (push e rlt)))
  rlt)
```

**Pull request merged**